test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -112,6 +112,11 @@ function Get-WebListenerUrl {
     [OutputType([Uri])]
     param (
         [switch]$Https,
+        [ValidateSet(
+            'Cert',
+            'Get',
+            '/'
```

Comment:
It semms we skip a description for '/' in Readme.md.

Reply:
`The default page will return a list of available tests.`

Reply:
Please clarify. Previously we used `Get-WebListenerUrl -Https` in tests - do we use by default '/'?

Reply:
we do return the `/` by default.

This is the scenario I included it for:

```powershell
$Testcases = @(
    @{
        Name           = 'Get Test'
        Test           = 'Get'
        ExpectedResult = 'Foo'
        Https          = $False
    }
    @{
        Name           = 'Cert test'
        Test           = 'Cert'
        ExpectedResult = 'Bar'
        Https          = $False
    }
    @{
        Name           = 'Default Test'
        Test           = '/'
        ExpectedResult = 'Baz'
        Https          = $False
    }
)

It "<Name>" -TestCases $TestCases {
    Param($Name, $Test, $ExpectedResult, $Https)
    $Url = Get-WebListenerUrl -Test $Test -Https:$Https
    Invoke-RestMethod | Should Match $ExpectedResult
}
```

Test is always passed, but a named test might not always be needed.

Reply:
Thanks! I think we need the explicit section.

Reply:
The `ValidetSet` already blocks them:

```powershell
Get-WebListenerUrl -Test ""
```
```none
Get-WebListenerUrl : Cannot validate argument on parameter 'Test'. The argument "" does not
belong to the set "Cert,Get,Home,/" specified by the ValidateSet attribute. Supply an argument
that is in the set and then try the command again.
At line:1 char:26
+ Get-WebListenerUrl -Test ""
+                          ~~
    + CategoryInfo          : InvalidData: (:) [Get-WebListenerUrl], ParameterBindingValidationEx
   ception
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Get-WebListenerUrl
```
```powershell
Get-WebListenerUrl -Test $Null
```
```none
Get-WebListenerUrl : Cannot validate argument on parameter 'Test'. The argument "" does not
belong to the set "Cert,Get,Home,/" specified by the ValidateSet attribute. Supply an argument
that is in the set and then try the command again.
At line:1 char:26
+ Get-WebListenerUrl -Test $Null
+                          ~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-WebListenerUrl], ParameterBindingValidationEx
   ception
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Get-WebListenerUrl
```

Reply:
Good!